bn_mp_radix_size_overestimate.c

```diff
@@ -0,0 +1,333 @@
+   3868050815ul, 2408285274ul,  427100031ul, 3594167896ul, 1065013491ul,
+   3441134334ul, 1200147405ul, 3195040475ul, 1343271782ul, 1429396043ul,
+    638097011ul, 1651143338ul,          0ul, 1492113523ul, 1495425743ul,
+    916355301ul, 2130026983ul, 1832592635ul, 1083835016ul, 2265160897ul,
+   3868050815ul, 3464916779ul, 2408285274ul, 2621526098ul,  427100031ul,
+   1703110503ul, 3594167896ul, 2513440227ul, 1065013491ul, 2686543564ul,
+   3441134334ul, 2560439235ul, 1200147405ul, 1309103046ul, 3195040475ul,
+       183550ul, 1343271782ul, 2148848508ul, 1429396043ul, 3785381171ul,
+    638097011ul, 1073048670ul, 1651143338ul, 3473298766ul,          0ul
+};
+
+/* TODO: This assumes the type "long long" exists. If it doesn't: use the tables for MP_16BIT */
+#elif ( (defined MP_28BIT) || (defined MP_31BIT) || (defined MP_32BIT)  || (defined MP_64BIT) )
+#define LTM_RADIX_SIZE_SCALE 61
+#ifdef MP_64BIT
+static const mp_digit logbases[65] = {
+#else
+static const mp_word logbases[65] = {
+#endif
+                      0ull,                    0ull,  2305843009213693952ull,
+    3654674702153732339ull,  4611686018427387904ull,  5354001665492895103ull,
+    5960517711367426291ull,  6473319721408181094ull,  6917529027641081856ull,
+    7309349404307464679ull,  7659844674706589055ull,  7976906213687625599ull,
+    8266360720581120243ull,  8532633055092329421ull,  8779162730621875046ull,
+    9008676367646627443ull,  9223372036854775808ull,  9425047617917838031ull,
+    9615192413521158631ull,  9795053960520401544ull,  9965687683920283007ull,
+   10127994423561913434ull, 10282749222901319551ull, 10430623713119086168ull,
+   10572203729794814195ull, 10708003330985790206ull, 10838476064306023373ull,
+   10964024106461197019ull, 11085005739835568998ull, 11201741516507896395ull,
+   11314519376860321395ull, 11423598928577852074ull, 11529215046068469760ull,
+   11631580915841357939ull, 11730890627131531983ull, 11827321386901076197ull,
+   11921035422734852583ull, 12012181624960263419ull, 12100896969734095496ull,
+   12187307757246061761ull, 12271530693133976959ull, 12353673836347420459ull,
+   12433837432775607386ull, 12512114650772492370ull, 12588592232115013503ull,
+   12663351069800359783ull, 12736466722332780120ull, 12808009872697200099ull,
+   12878046739008508147ull, 12946639442816362188ull, 13013846340199484158ull,
+   13079722320071570371ull, 13144319073519717325ull, 13207685337486676934ull,
+   13269867115674890971ull, 13330907879180520702ull, 13390848749049262950ull,
+   13449728662674133884ull, 13507584525721590347ull, 13564451351070528819ull,
+   13620362386074015347ull, 13675349229302017118ull, 13729441937791546026ull,
+   13782669125715645774ull, 13835058055282163712ull
+};
+#else
+#  ifdef _MSC_VER
+#    pragma message("mp_estimate_string_size: unknown or undefined MP_xBIT type")
+#  else
+#    warning "mp_estimate_string_size: unknown or undefined MP_xBIT type"
+#  endif
+#endif
+#if 0
```

Comment:
#if 0? Why add dead code?

Reply:
Because: 
 - it was quite fuddly to compute, didn't want to dump it so easily.
 - it is also not "dead code" in its strictest meaning because it must be used for 64-bit `int`s.
 - it shows some optimism.
 - it is futureproof until 2105. According to Nkasiogi Koslowsky who will have invented a time-machine in 2091, went to 2106 and nicked a mobile. At least that's what she told me but I doubt it because she said that it was an Apple.

But I can delete it, if you insist?

Reply:
Hmm but why is it then not enabled for 64 bit ints. How do you detect int size?

Reply:
> How do you detect int size?

I don't think that there will be 64-bit large `int`s in the near future and I don't need the size elswhere in that function, so I just don't check the size of an `int` and wait until someone complains.